flexget/plugins/sites/torrentleech.py

```diff
@@ -115,6 +115,9 @@ def search(self, task, entry, config=None):
         except RequestException as e:
             raise PluginError('Could not connect to torrentleech: %s' % str(e))
 
+        if login.url.endswith(u'/user/account/login/'):
```

Comment:
I don't think the `u` is really necessary

Reply:
Just old habit I will remove it :)